### PR TITLE
CoeboCanteen v1.0.0

### PR DESCRIPTION
**Pull request by Goodjooy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120196)

### CODE REVIEW

**Comment by Goodjooy** [comment]
> @Goodjooy please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by Goodjooy** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

Does the installer have silent install switches?

File: manifests/c/CeobeCanteen/DesktopApp/1.0.1/CeobeCanteen.DesktopApp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PackageIdentifier: CeobeCanteen.DesktopApp
+PackageVersion: 1.0.1
+InstallerLocale: zh-CN
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
```
Comment (stephengillie) [inline manifests/c/CeobeCanteen/DesktopApp/1.0.1/CeobeCanteen.DesktopApp.installer.yaml:13]:
```suggestion
- Silent: /S
- SilentWithProgress: /S
```

**Comment by stephengillie** [comment]
Hi @Goodjooy,

Manual Validation ended with:
> 7:59:46 PM:  > 2023-09-19 19:59:44.845 [FAIL] Installer failed security check. Url: https://github.com/Enraged-Dun-Cookie-Development-Team/Ceobe-Canteen-Electron/releases/download/tauri-v1.0.1-build.39.1/CeobeCanteen_1.0.1_x64-setup.exe Result: 0x80004005 

The package did not pass a security check. It's similar to a Defender scan. This might be a false positive, and we can retry tomorrow.

**Comment by stephengillie** [comment]
Hi @Goodjooy,

This is a retry. The package has failed a security check, and also a Defender scan:
image

This might not be a false positive. Should the developer be notified?

**Pull request closed without merging**